README.md

```diff
@@ -4,7 +4,83 @@
 # HistoLab
 
-### NOTE: WORK IN PROGRESS PROJECT, ANY KIND OF CONTRIBUTION IS APPRECIATE 
+### NOTE: WORK IN PROGRESS PROJECT
+
+## Table of Contents
+
+* [Motivation](#motivation)
+ 
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+  * [Quickstart](#quickstart)
+* [Versioning](#versioning)
+* [Authors](#authors)
+* [License](#license)
+* [Roadmap](#roadmap)
+* [Acknowledgements](#acknowledgements)
+* [References](#references)
+* [Contribution guidelines](#contribution-guidelines)
+
+
+## Motivation
+The histo-pathological analysis of tissue sections is the gold standard to assess the presence  and understand the nature of many complex diseases, such as tumors. 
+In daily practice, pathologists usually perform microscopy examination of tissue slides considering a limited number of regions and the clinical evaulation relies on several factors such as nuclei morphology, cell distribution, and color (staining): this process is time consuming, could lead to information loss, and suffers from inter-observer variability.
+
+The advent of digital pathology is changing the way patholgists work and collaborate, and has opened the way to a new era in computational pathology. In particular, histopathology is expected to be at the centre of the AI revolution in medicine [1], prevision supported by the increasing success of deep learning applications to digital pathology.
+
+Whole Slide Images (WSIs), namely the translation of tissue slides from glass to digital format, are a great source of information from both a medical and a computational point of view. WSIs can be coloured with different staining technicques (e.g. H&E or IHC), and are usually very large in size (up to several GB per slide). Because of WSIs typical pyramidal structure, images can be retrieved at different magnification factors, providing a further layer of information beyond color.
+
+However, processing WSIs is far from being trivial. First of all, WSIs can be stored in different proprietary formats, according to the scanner used to digitalized the slides, and a standard protocol is still missing. WSIs can also present artifacts, such as shadows, mold, or annotations (pen marks) that are not useful. Moreover, giving their dimensions, it is not possible to process a WSI all at once, or, for example, to feed a neural network: it is necessary to crop smaller regions of tissues (tiles), which in turns require a tissue detection step.  
+
+The aim of this project is to provide a tool for WSI processing in a reproducible environment to support clinical and scientific research. Histolab is designed to handle WSIs, automatically detect the tissue, and retrieve informative tiles, and it can thus be integrated in a deep learning pipeline.
+
+See the detailed documentation [here]().
```

Comment:
also the link is missing, I know... but by default it redirects here https://github.com/MPBA/histolab/blob/write_readme which gives 404

Reply:
should I remove it?

Reply:
until we have the docs hosted somewhere, yes